runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -155,30 +155,31 @@ TR::Node *
 J9::TransformUtil::generateArrayOffsetTrees(TR::Compilation *comp, TR::Node *indexNode, TR::Node *elementSizeNode, int32_t elementSize, bool useShiftOpCode)
    {
    TR::Node *offsetNode = indexNode->createLongIfNeeded();
+   if (elementSizeNode)
+      elementSizeNode = elementSizeNode->createLongIfNeeded();
```

Comment:
Let's create a local variable for `elementSizeNode` for better readability

Reply:
Not sure if something like this is better... you still have to check for null-ness, and I can't think of a better name than `elementSizeNode`, anything in mind?

```
   TR::Node *strideNode = elementSizeNode;
   if (strideNode)
      strideNode = strideNode->createLongIfNeeded();
```

Reply:
yea, I think stride is a good name for the variable. Let's replace all uses of `elementSizeNode` with `strideNode` after this line.